pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -65,6 +65,7 @@ const (
 	AggregatedClusterReaderRoleName = "system:openshift:aggregate-to-cluster-reader"
 	SelfProvisionerRoleName         = "self-provisioner"
 	BasicUserRoleName               = "basic-user"
+	BasicSCCUserRoleName            = "basic-scc-user"
```

Comment:
Will need to update the bootstrap policy test YAML:

`UPDATE_BOOTSTRAP_POLICY_FIXTURE_DATA=true hack/test-go.sh pkg/cmd/server/bootstrappolicy`